frontend/src/pages/Admin/Courses.js

```diff
@@ -1,24 +1,25 @@
-import React, { useEffect, useState } from 'react';
-import { Button } from '@trussworks/react-uswds';
+import React, { useCallback, useEffect, useState } from 'react';
+import { Button, Grid, GridContainer } from '@trussworks/react-uswds';
 import CsvImport from './components/CsvImport';
 import { getCourses } from '../../fetchers/courses';
+import Container from '../../components/Container';
+import CourseList from './CourseList';
+import CourseAdd from './CourseAdd';
 
 function Courses() {
   const [courses, setCourses] = useState();
   const primaryIdColumn = 'course name';
   const typeName = 'courses';
   const apiPathName = 'courses';
 
-  useEffect(() => {
-    async function get() {
-      const response = await getCourses();
-      setCourses(response);
-    }
+  const refresh = useCallback(async () => {
```

Comment:
Should we be using useEffect here? Why the change?

Reply:
`refresh` is a function that is provided to a child component, so it should be memoized with `useCallback`.